src/payment/checkout/Checkout.jsx

```diff
@@ -182,6 +182,15 @@ class Checkout extends React.Component {
     const shouldDisplayStripePaymentForm = !loading && enableStripePaymentProcessor && options.clientSecret;
     const shouldDisplayCyberSourcePaymentForm = !loading && !enableStripePaymentProcessor;
 
+    let stripePromise;
+    if (shouldDisplayStripePaymentForm) {
```

Comment:
I'm curious. how this behaved locally - because `shouldDisplayStripePaymentForm` will only be true if it's not loading anymore and we have flag is true + client secret from capture-context. Does creating the Stripe instance only when this is true, does it delay Stripe's loading skeleton for the card form?

Reply:
Based on our conversation this morning, I will merge this branch and look into its performance on stage, and will revert if need be later 👍 Thank you for raising!